jobs/cjd_sample/spec/quickstart_spec.rb

```diff
@@ -0,0 +1,39 @@
+# Copyright 2017 Google, Inc
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require "rspec"
+require "rspec/retry"
+require "google/apis/jobs_v2"
```

Comment:
this shouldn't be required.

Reply:
It seems I could not run the test via "bundle exec rspec" without these 3 lines.

Reply:
What error are you seeing?
Reason why I recommend not including it is your quickstart includes this requirement.